---
/*/diving_node:
  ros_parameters:
    view_rate: 1/10
    model_rate: 1/10
    controller_rate: 1/10
    vbs_pid_kp: 40.0
    vbs_pid_ki: 5.0
    vbs_pid_kd: 1.0
    vbs_pid_kaw: 1.0
    vbs_u_neutral: 50.0
    vbs_u_min: 0.0
    vbs_u_max: 100.0
    lcg_pid_kp: 40.0
    lcg_pid_ki: 5.0
    lcg_pid_kd: 1.0
    lcg_pid_kaw: 1.0
    lcg_u_neutral: 50.0
    lcg_u_min: 0.0
    lcg_u_max: 100.0
    tv_pid_kp: 2.5
    tv_pid_ki: 0.25
    tv_pid_kd: 0.5
    tv_pid_kaw: 1.0
    tv_u_neutral: 0.0
    tv_u_min: -0.122173
    tv_u_max: 0.122173
    yaw_pid_kp: 2.5
    yaw_pid_ki: 0.25
    yaw_pid_kd: 0.5
    yaw_pid_kaw: 1.0
    yaw_u_neutral: 0.0
    yaw_u_min: -0.122173
    yaw_u_max: 0.122173
